.rhcicd/clowdapp-backend.yaml

```diff
@@ -215,6 +215,20 @@ objects:
               optional: true
         - name: KESSEL_DOMAIN
           value: ${KESSEL_DOMAIN}
+        - name: QUARKUS_OIDC_CLIENT_AUTH_SERVER_URL
+          value: ${OIDC_ISSUER}
+        - name: QUARKUS_OIDC_CLIENT_CLIENT_ENABLED
+          value: ${OIDC_CLIENT_ENABLED}
+        - name: QUARKUS_OIDC_CLIENT_CLIENT_ID
+          valueFrom:
+            secretKeyRef:
+              name: kessel-authentication
```

Comment:
Shoudn't we rename those keys to something more generic like `user-access-authentcation` (because it's the name of Rbac app in the console and it could be more stable than the internal team name?).
Specially because `relations-api` should be remove in a near future.

Reply:
great, thank you :)